cypress/support/commands.js

```diff
@@ -11,3 +11,19 @@ Cypress.Commands.add('loginAs', (role) => {
     window.localStorage.setItem('ACCESS_TOKEN', `userId=1&${USER_ROLES_MAP[role]}`)
 })
 
+Cypress.Commands.add('login', (user, role) => {
+
+    cy.visit('/')
+        cy.get('.Select.not-valid').should('not.visible')
+        cy.get('[type="submit"]').click()
+        cy.get('.Select.not-valid').should('be.visible')
+
+        //Login with "User" role
```

Comment:
Do not add such comments, because the user will not always be logged in with "User" role as you give role as the parameter in this function

Reply:
Yep I have changed that already, but good catch!!  👍